src/components/onboardinng-setup-pin/onboarding-setup-pin.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+
+import {StyleSheet, View} from 'react-native';
+
+import {Color, getColor} from '@app/colors';
+import {NumericKeyboard} from '@app/components/pin/numeric-keyboard';
+import {PopupContainer, Spacer, Text} from '@app/components/ui';
+import {I18N} from '@app/i18n';
+
+interface OnboardingSetupPinProps {
+  onKeyboard: (value: number) => void;
+  pin: string;
+}
+
+export const OnboardingSetupPin = ({
+  onKeyboard,
+  pin,
+}: OnboardingSetupPinProps) => {
+  return (
+    <PopupContainer style={page.container} testID="onboarding-setup-pin">
+      <Text t4 i18n={I18N.onboardingSetupPinSet} style={page.title} />
+      <Text
+        t11
+        color={Color.textBase2}
+        center
+        i18n={I18N.onboardingSetupPinProjectWallet}
+        i18params={{first: '\u00A0', second: '\u00A0'}}
```

Comment:
why?????

Reply:
I thought i18n wouldn't accept them and didn't even try, sorry, I fix it